script/dryrun

```diff
@@ -39,7 +39,8 @@ SKIP = {
     "sql/telemetry/fxa_oauth_events_v1/query.sql",
     "sql/search_derived/search_clients_last_seen_v1/init.sql",
     "sql/search_derived/search_clients_last_seen_v1/query.sql",
-    "sql/search/search_rfm/view.sql",
+    "sql/search/search_rfm/init.sql",
+    "sql/search/search_rfm/query.sql",
```

Comment:
This should move to search_derived, add a version, and be exposed by a `sql/search/search_rfm/view.sql`

```suggestion
    "sql/search_derived/search_rfm_v1/query.sql",
```